src/renderer/dx/DxRenderer.cpp

```diff
@@ -424,8 +424,7 @@ void DxEngine::_ComputePixelShaderSettings() noexcept
         {
             switch (_chainMode)
             {
-            case SwapChainMode::ForHwnd:
-            {
+            case SwapChainMode::ForHwnd: {

```

Comment:
this is a really dumb thing that is happening with clang 9.0 -- VS will corrupt the format now until we fix